src/Services/EncounterService.php

```diff
@@ -107,31 +104,47 @@ public function getEncountersForPatient($pid)
                        LEFT JOIN openemr_postcalendar_categories as opc
                        ON opc.pc_catid = fe.pc_catid
                        LEFT JOIN facility as fa ON fa.id = fe.billing_facility
-                       WHERE pid=?
+                       WHERE pid=? and fe.uuid=?
```

Comment:
just to make this more clear for future developers, would prepend fe. to the pid:
```
WHERE fe.pid=? and fe.uuid=?
```